spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'web_helpers'
+
+RSpec.feature "Sessions",  type: :feature do
+
+  scenario "signs up the user", :type => :feature do
+    sign_up
+    page.should have_content("Welcome! You have signed up successfully.")
+  end
+
+  scenario "signs in the user", :type => :feature do
+    sign_up
+    visit "/users/sign_out"
```

Comment:
Thanks Dan , doing things in a hurry I forgot to mention the link .It's all setup now.